integration_test.go

```diff
@@ -0,0 +1,231 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"math/rand"
+	"os"
+	"os/exec"
+	"testing"
+	"text/template"
+	"time"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/api/iterator"
+)
+
+const (
+	logFilePath    = "./logfile.log"
```

Comment:
Can this be redefined to "./" + logFileName?

Reply:
Done